### ISSUE TITLE
Fetching kernel GPG key ... Failed

### ISSUE THREAD

**Issue by BertrandBataille** [rickard2/grsecurity-Debian-Installer#19]
I'm trying to use this installer on a brand new install of Debian Jessie 8.1.0 but it just fails as follows:

==> Checking current versions of grsecurity ...
######################################################################## 100.0%
==> 1. grsecurity version 3.1 for kernel 3.2.69, revision 201507251415 (stable version)
==> 2. grsecurity version 3.1 for kernel 3.14.48, revision 201507261203 (stable version)
==> 3. grsecurity version 3.1 for kernel 4.1.3, revision 201507261202 (testing version)
==> Please make your selection: [1-3]: 2
==> Remove build tools after install? (build-essential bin86 kernel-package libncurses5-dev zlib1g-dev gcc-4.9-plugin-dev bc): [y/N] 
==> Installing grsecurity stable version 3.1 using kernel version 3.14.48 ... 
==> Fetching kernel GPG key ... Failed

**Comment by rickard2** [comment]
You might be missing the gnupg tools, try installing them with `sudo apt-get install gnupg` and try again. 

I will try update the install script to also install this tool

**Comment by BertrandBataille** [comment]
No, that's definitely not it. I had just finished generating my own GnuPG key minutes before.

**Comment by cthbleachbit** [comment]
gpg should do `gpg --recv-keys` from the keyserver first before verifying signatures
